test/test-v8debugapi.js

```diff
@@ -43,6 +45,67 @@ function stateIsClean(api) {
   return true;
 }
 
+function validateVariable(variable) {
+  if (variable.name) {
+    assert.equal(typeof variable.name, 'string');
+  }
+  if (variable.value) {
+    assert.equal(typeof variable.value, 'string');
+  }
+  if (variable.type) {
+    assert.equal(typeof variable.type, 'string');
+  }
+  if (variable.members) {
+    variable.members.forEach(validateVariable);
+  }
+  if (variable.varTableIndex) {
+    assert.ok(Number.isInteger(variable.varTableIndex) &&
```

Comment:
These checks are certainly exercised, with the exception of `type`. We do not populate the `type` property of a variable. I don't think there is harm on asserting on what we want the type to be if we decide to start populating it in the future.